examples/ledger/ledger.import

```diff
@@ -1,27 +1,33 @@
 #!/usr/bin/env python3
```

Comment:
Why rename this python script `ledger.import` doesn't `import.py` make more sense? Also, the directory in which the script is should be in the python path, thus the form `from importers import utrade` should work without having messing with the path, which IMHO is better style (at the price of a `__init__.py` in the `importers` directory.

Reply:
ledger.import, import.py... I don't have a strong opinion. I have something like five ledger.* files myself. Either way. I'll leave it here for now, happy to rename later.

About the path: no strong opinion either. The reason I flattened it is to try to make it as simple as possible. I want the example to keep things so obvious there are the least amount of questions. We can change it.